flake.nix

```diff
@@ -1,17 +1,18 @@
 {
-  description =
-    "llvm-codegen: LLVM code generation using Haskell";
+  description = "llvm-codegen: LLVM code generation using Haskell";
   inputs = {
     np.url = "github:nixos/nixpkgs?ref=haskell-updates";
     fu.url = "github:numtide/flake-utils?ref=master";
     ds.url = "github:numtide/devshell?ref=master";
+    nf.url = "github:numtide/nix-filter?ref=master";
   };
-  outputs = { self, np, fu, ds }:
+  outputs = { self, np, fu, ds, nf, ... }@inputs:
     with np.lib;
     with fu.lib;
     eachSystem [ "x86_64-linux" ] (system:
       let
         ghcVersion = "902";
+        llvmVersion = 14;
```

Comment:
Nice. But for non-nix users we still need to update it in Setup.hs as well, right?

Reply:
Correct